test/BreakInvariantBounty.test.js

```diff
@@ -17,82 +17,95 @@ const sendReward = async (from, to, value) => ethSendTransaction({
 
 const reward = new web3.BigNumber(web3.toWei(1, 'ether'));
 
-contract('BreakInvariantBounty', function ([_, owner, researcher, nonTarget]) {
-  context('against secure contract', function () {
-    beforeEach(async function () {
-      this.bounty = await SecureInvariantTargetBounty.new({ from: owner });
-    });
-
-    it('can set reward', async function () {
-      await sendReward(owner, this.bounty.address, reward);
-
-      const balance = await ethGetBalance(this.bounty.address);
-      balance.should.be.bignumber.equal(reward);
-    });
-
-    context('with reward', function () {
-      beforeEach(async function () {
-        const result = await this.bounty.createTarget({ from: researcher });
-        const event = expectEvent.inLogs(result.logs, 'TargetCreated');
-
-        this.targetAddress = event.args.createdAddress;
-
-        await sendReward(owner, this.bounty.address, reward);
-
-        const balance = await ethGetBalance(this.bounty.address);
-        balance.should.be.bignumber.equal(reward);
-      });
+contract('BreakInvariantBounty', function ([_, owner, researcher, anyone, nonTarget]) {
+  beforeEach(async function () {
+    this.bounty = await BreakInvariantBountyMock.new({ from: owner });
+  });
 
-      it('cannot claim reward', async function () {
-        await assertRevert(
-          this.bounty.claim(this.targetAddress, { from: researcher }),
-        );
-      });
-    });
+  it('can set reward', async function () {
+    await sendReward(owner, this.bounty.address, reward);
```

Comment:
I don't like this `sendReward` name. What would you think of dropping the function and just calling web3.eth.sendTransaction?

Reply:
Opened https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1358